test/entity/entity_test.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 const (
-	sleepForFlush = 180 * time.Second
+	sleepForFlush = 240 * time.Second
```

Comment:
4 minutes sleep makes sense based off waiting for agent/FluentBit/GetQueryResults/buffer